R/replay-html.r

```diff
@@ -113,6 +137,14 @@ replay_html.recordedplot <- function(x, name_prefix, obj_id, ...) {
   paste0("<img src='", escape_html(path), "' alt='' width='540' height='400' />")
 }
 
+#' @export
+replay_html.knit_asis <- function(x, name_prefix, obj_id, ...) {
+  output <- paste0("</pre><div class='well knit_asis'>", x, "</div><pre class=\"example\">")
```

Comment:
We need to cross-reference this with the example starting and ending tags to make sure it stays consistent.